### DISCUSSION TITLE
How do I make `duckbox` output show more than 40 rows?

### DISCUSSION THREAD

**Discussion by pkese** [duckdb/duckdb#10562] [Q&A]
I have a large monitor and outputs of about 60 rows, which makes it really annoying, that duckdb shows only 40 rows at at time.

Is it possible to set the output to more than 40 rows?

**Comment by Tishj** [comment]
Yes, see the list of options on <https://duckdb.org/docs/api/cli/dot_commands.html>

**Reply by pkese** [reply]
Aha, `.maxrows  70` fixed it. Thanks.
